.travis/publish-gh-pages.sh

```diff
@@ -3,13 +3,6 @@
 VERSION=`node -e "console.log(require('./package.json').version)"`
 TMP_GH_PAGES_DIR=.gh-pages-demo
 
-# We only want to deploy to gh-pages from "master"
-if [ "${TRAVIS_BRANCH}" != "master" ]
-then
-    echo "Skipping gh-pages publish for branch ${TRAVIS_BRANCH}"
-    exit 0
-fi
-
```

Comment:
You still want this here, I think what you want to do is make the publish conditional as well, making sure it's not the bump commit, and moving it to happen before the version bump happens. I think @sandersky was gonna update `ember-frost-bunsen` to let it's demo work again with the latest `pr-bumper` configs, not sure if he got the chance to or not yet.